spec/forms/webauthn_visit_form_spec.rb

```diff
@@ -90,7 +99,13 @@
 
         it 'returns FormResponse with success: false with an unrecognized error' do
```

Comment:
Can we add test coverage for the new branching `if`, since there are now two paths?